### PR TITLE
Updated with info from bug.

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Comment by nbedarkar** [comment]
LGTM

**Review by evinaku — DISMISSED** [review]

File: modules/ROOT/pages/cloudhub-api.adoc [hunk 1]
```diff
@@ -126,6 +126,48 @@ Here is an example of data received in JSON format in response to a request to g
 }
 ----
 
+== Rate Limits 
+
+The CloudHub management API enforces usage limits.
+These limits don't apply to CloudHub workers.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-api.adoc:133]: Should we add something here about "global" and perhaps imply the following two subtopics (exceptions)?

Comment (kbostrom) [reply modules/ROOT/pages/cloudhub-api.adoc:133]: Wow, this is vexatious. We could easily cover this in three bullets. Sigh.

Comment (kbostrom) [reply modules/ROOT/pages/cloudhub-api.adoc:133]: And these tables are butt-ugly.

File: modules/ROOT/pages/cloudhub-api.adoc [hunk 2]
```diff
@@ -126,6 +126,48 @@ Here is an example of data received in JSON format in response to a request to g
...
+|EU .2+| 25 requests per second
+|Government Cloud
+|===
+
+
+=== Exceptions to the Global Rate Limits
+
+The following table lists the exceptions to the <<Global Rate Limits for CloudHub APIs>>. 
+
+When the app reaches the rate limit, CloudHub returns the status code.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-api.adoc]:
```suggestion
When the app reaches the rate limit, CloudHub returns the status codes shown here.
```

File: modules/ROOT/pages/cloudhub-api.adoc [hunk 3]
```diff
@@ -137,22 +179,17 @@ When you call the REST APIs, the following status codes are returned:
...
-** `v2/applications/{domain}/deployments/{deploymentId}/logs`
-** `v2/applications/{domain}/instances/{instanceId}/logs`
-* The following endpoint has a limit of one call per minute for each application:
-** `v2/applications/{domain}/instances/{instanceId}/log-file`
+|429 a|The operation was unsuccessful due to reaching the rate limit.
 |500 |The operation was unsuccessful.
 
 See the HTTP body for details.
-|===
+|503 |The operation was unsuccessful due to a temporary condition, such as server overload or rate limit.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-api.adoc]:
```suggestion
|503 |The operation was unsuccessful due to a temporary condition, such as server overload or reaching a rate limit.
```

**Review by kbostrom — DISMISSED** [review]
Edited

**Review by kbostrom — APPROVED** [review]
Reapproval

**Pull request merged**